test/FunctionalTests/Microsoft.OData.Edm.Tests/ExtensionMethods/TargetHelperTests.cs

```diff
@@ -0,0 +1,216 @@
+﻿//---------------------------------------------------------------------
+// <copyright file="TargetHelperTests.cs" company="Microsoft">
+//      Copyright (C) Microsoft Corporation. All rights reserved. See License.txt in the project root for license information.
+// </copyright>
+//---------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.OData.Edm.Tests.ExtensionMethods

```

Comment:
>ExtensionMethods [](http://example.com/codeflow?start=36&length=16)

the namespace looks weird. We don't need the "ExtensionMethods" in the namespace, right?

Reply:
Seems other files in the ExtensionMethods folder have it 
https://github.com/OData/odata.net/blob/d82dcfa51f07c1cd3da0326a4ff950a6827ccc9d/test/FunctionalTests/Microsoft.OData.Edm.Tests/ExtensionMethods/EdmElementComparerTests.cs#L10